sample_code_submission/model.py

```diff
@@ -38,34 +69,40 @@ def __init__(self, get_train_set=None, systematics=None):
         Returns:
             None
         """
-
-        self.systematics = systematics
-
         self.get_train_set = get_train_set
-
+        self.systematics = systematics
         self.re_train = True
 
-        from boosted_decision_tree import BoostedDecisionTree
+        if XGBOOST:
```

Comment:
So you brought added the 3 models from test_model ?

Reply:
I haven't changed anything in this file. It is the test model on codabench